components/snackbar.css

```diff
@@ -0,0 +1,44 @@
+.snackbar{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    width:27rem;
+    color:var(--text-white-color);
+    padding:1rem;
+    border-radius: 4px;
```

Comment:
Since there is a scope of using same border radius at different places as well, I suggest using a css variable here

Reply:
I have just use it at 2 places.so, will not need any variable